services/appveyor/appveyor-tests.tester.js

```diff
@@ -56,14 +57,14 @@ t.create('Test status with custom labels')
 
 t.create('Test status with compact message and custom labels')
   .timeout(10000)
-  .get('/NZSmartie/coap-net-iu0to.json', {
-    qs: {
+  .get(
+    `/NZSmartie/coap-net-iu0to.json?${queryString.stringify({
```

Comment:
The old version produced `?compact_message=`; the new one produces `?compact_message` which is what we are supposed to support.